core/src/test/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -143,7 +143,6 @@ private void slowDownRecovery(ByteSizeValue shardSize) {
                 .setTransientSettings(Settings.builder()
                                 // one chunk per sec..
                                 .put(RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC, chunkSize, ByteSizeUnit.BYTES)
-                                .put(RecoverySettings.INDICES_RECOVERY_FILE_CHUNK_SIZE, chunkSize, ByteSizeUnit.BYTES)
```

Comment:
I think the reason why I did like this in the past is that I didn't want the recovery to completely stall because it wants to transfer 512Kb with a rate of 5b per sec , which will make it sleep for a long time and the test times out.  Maybe we should change the sleep logic to sleep with smaller byte sizes (and iterate).